android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -448,6 +456,29 @@ private void purchasePackage(final String packageIdentifier,
         );
     }
 
+    private void purchaseSubscriptionOption(final String productIdentifier,
+                                            final String optionIdentifier,
+                                            final String googleOldProductId,
+                                            @Nullable final Integer googleProrationMode,
+                                            @Nullable final Boolean googleIsPersonalizedPrice,
+                                            final Result result) {
+        // TODO: Convert this somehow
+        GoogleProrationMode googleProrationModeEnum = null;
```

Comment:
you can check out the canMakePayments `BillingFeature` conversion, might be similar..